build/stf-run-ci/defaults/main.yml

```diff
@@ -21,3 +21,10 @@ sg_core_image_tag: latest
 sg_bridge_image_tag: latest
 prometheus_webhook_snmp_image_tag: latest
 namespace: service-telemetry
+
+# used when building images to default to correct version branch for STF subcomponents per STF version
+version_branches:
```

Comment:
Just to check my understanding - would this file will be adjusted on a stable-* branch so  that it mentions the corresponding branches in the other repos?

Reply:
I just read PR#249 and yes, it seems I understood correctly :)